cloudbuild.yaml

```diff
@@ -7,12 +7,10 @@ steps:
   dir: &dir '/workspace/solo-kit'
 
 # prepare-workspace to set up the project so it can be built and tested
-- name: 'gcr.io/$PROJECT_ID/prepare-go-workspace:0.2.2'
+- name: 'gcr.io/$PROJECT_ID/prepare-go-workspace:0.4.28'
```

Comment:
cloud-builders bumps were needed to fix the [gke auth plugin](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke) errors in ci